File: VB/VB/GoogleTranslatorProvider/Module.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp

Namespace GoogleTranslatorProvider
    Public NotInheritable Partial Class GoogleTranslatorProviderModule
        Inherits ModuleBase
        Public Sub New()
            InitializeComponent()
            DevExpress.ExpressApp.Utils.TranslatorProvider.RegisterProvider(New GoogleTranslatorProvider())
        End Sub
    End Class
End Namespace

File: CS/GoogleTranslatorProvider/Module.cs
using System;
using DevExpress.ExpressApp;

namespace GoogleTranslatorProvider {
    public sealed partial class GoogleTranslatorProviderModule : ModuleBase {
        public GoogleTranslatorProviderModule() {
            InitializeComponent();
            DevExpress.ExpressApp.Utils.TranslatorProvider.RegisterProvider(new GoogleTranslatorProvider());
        }
    }
}
